File: aiui/src/main/java/com/iflytek/vtncaetest/ContextHolder.java
package com.iflytek.vtncaetest;

import android.app.Application;
import android.content.Context;

/**
 * 存储Application context
 */
public class ContextHolder extends Application {

    private static Context context;

    @Override
    public void onCreate() {
        super.onCreate();
        context = getApplicationContext();
    }

    /**
     * 获取全局上下文
     */
    public static Context getContext() {
        return context;
    }
}

File: xtts/src/main/java/com/iflytek/aikitdemo/ContextHolder.kt
package com.iflytek.aikitdemo

import android.content.Context
import kotlin.properties.Delegates

/**
 * @Desc:
 * @Author leon
 * @Date 2023/2/24-11:01
 * Copyright 2023 iFLYTEK Inc. All Rights Reserved.
 */
class ContextHolder{

    companion object {
        var CONTEXT: Context by Delegates.notNull()
    }

//    override fun attachBaseContext(base: Context?) {
//        super.attachBaseContext(base)
//        MultiDex.install(this)
//    }
//    override fun onCreate() {
//        super.onCreate()
//        //仅仅作为测试性能用
//        connectMemoryStatsPipe()
//    }
}